src/battleship/Battleship.java

```diff
@@ -63,88 +79,115 @@ public int colNum(String coordinate) {
-        Board board = new Board();
-        Board hiddenBoard = new Board();
-        List<Ship> ships = new ArrayList<>(Arrays.asList(
-            new Ship(5, "Aircraft Carrier"),
-             new Ship(4, "Battleship"),
-             new Ship(3, "Submarine"),
-             new Ship(3, "Cruiser"),
-            new Ship(2, "Destroyer")
+
+        boolean playerOneWin = false;
+        boolean playerTwoWin = false;
+
+//        while (playerOneWin != true || playerTwoWin != true) {
+//
+//            Optional<String> coordinate = Board.parseCoordinate(scanner.nextLine());
+//
+//            if (coordinate.isEmpty()) {
+//                System.out.println("Invalid input. Try again.");
+//                continue;
+//            }
+//
+//            int row = rowNum(coordinate.get());
+//            int col = colNum(coordinate.get());
+//
+//            if (row < 0 || col < 0) {
+//                System.out.println("Invalid location. Try again.");
+//                continue;
+//            }
+//
+//            try {
+//                if (board.isShip(row, col)) {
+//                    hiddenBoard.setIndex(row, col, 'X');
+//                    board.setIndex(row, col, 'X');
+//                    hiddenBoard.printField();
+//                    for (Ship ship : ships) {
+//                        board.isSunken(ship);
+//                    }
+//                    boolean sank = false;
+//                    for (Ship type : ships) {
+//                        if (type.isSunken()) {
+//                            sank = true;
+//                            ships.remove(type);
+//                            break;
+//                        }
+//                    }
+//                    if (sank) {
+//                        System.out.println("You sank a ship! Specify a new target:");
+//                    } else {
+//                        System.out.println("You hit a ship! Try again:");
+//                    }
+//                    oCounter--;
+//                } else if (board.isEmpty(row, col)) {
+//                    hiddenBoard.setIndex(row, col, 'M');
+//                    board.setIndex(row, col, 'M');
+//                    hiddenBoard.printField();
+//                    System.out.println("You missed. Try again:");
+//                } else if (board.isHit(row, col)) {
+//                    hiddenBoard.printField();
+//                    System.out.println("You already hit this one. Try again:");
+//                } else if (board.isMiss(row, col)) {
+//                    hiddenBoard.printField();
+//                    System.out.println("You already missed this one. Try again:");
+//                }
+//            } catch (IllegalArgumentException | ArrayIndexOutOfBoundsException e) {
+//                System.out.println("Error! You entered the wrong coordinates! Try again:");
+//            }
+//        }
+//        System.out.println("You sank the last ship. You won. Congratulations!");
```

Comment:
You would not normally commit commented code. You can always keep commented code in a git-stash or on another branch. In a team, merging it back to master is just burdening the rest of the team with code that is not yet ready.